src/components/widgets/accounts-cash-projection/accounts-cash-projection.tmpl.html

```diff
@@ -5,56 +5,86 @@
 
     <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
 
-    <!-- Buttons displayed on the lower  -->
     <div class="bottom-buttons" align="right">
       <button class="btn btn-default" ng-click="initSettings()">Cancel</button>
       <button class="btn btn-warning" ng-click="updateSettings()">Save</button>
     </div>
   </div>
 
-  <!-- Content Panel -->
+  <!-- Widget content -->
   <div ng-hide="widget.isEditMode">
-    <!-- Transactions list -->
-    <transactions-list ng-if="trxList.display" transactions="trxList.transactions" total-due="trxList.totalDue" currency="widget.metadata.currency" total-records="trxList.totalRecords" on-hide="trxList.hide()" on-page-changed="trxList.fetch(page)" on-update-expected-date="trxList.updateExpectedDate(trxId, date)"></transactions-list>
+    <transactions-list ng-if="trxList.display" transactions="trxList.transactions" currency="widget.metadata.currency" total-records="trxList.totalRecords" resources-type="trxList.resources" on-hide="trxList.hide()" on-page-changed="trxList.fetch(page)" on-update-expected-date="trxList.updateExpectedDate(trxId, date)" on-change-resources="trxList.changeResourcesType(resourcesType)" on-delete-transaction="trxList.deleteTransaction(resourcesType, trxId)"></transactions-list>
 
-    <!-- Chart -->
     <div ng-hide="trxList.display">
-      <!-- Set KPI target threshold panel -->
       <chart-threshold widget="widget" chart-promise="chartPromise" kpi-create-label="chartThresholdOptions.label" on-complete="widget.format()"></chart-threshold>
 
       <!-- Chart -->
       <div id="{{chartId()}}" class="cash-projection-chart"></div>
 
-      <!-- Simulation mode box -->
-      <div class="offset-box" ng-show="simulationMode">
-        <h3>Offset your cash projection</h3>
+      <!-- Bottom buttons -->
+      <div class="text-left">
+        <button class="btn btn-sm btn-default" ng-click="trxList.showAll()" title="See all due invoices and bills, and forecast">
+          <i class="fa fa-list" />
+          See all transactions
+        </button>
+        <button class="btn btn-sm btn-default" ng-click="addForecastPopup.show()" title="Create a new forecast transaction">
+          <i class="fa fa-pencil" />
+          Add forecast
+        </button>
+      </div>
+    </div>
+
+    <div ng-show="widget.demoData" common-data-not-found />
+  </div>
+
+  <!-- Add forecast popup -->
```

Comment:
Good to move this to it's own template, it only interacts with the `addForecastPopup` scope, so that could easily be a component scope I think.